cookbooks/aws-parallelcluster-environment/files/default/ec2_udev_rules/ec2_dev_2_volid.py

```diff
@@ -132,5 +98,54 @@ def main():
     print(volume_id)
 
 
+def main():
+    syslog.syslog("Starting ec2_dev_2_volid.py script")
+    # Get dev
+    try:
+        dev = str(sys.argv[1])
+        validate_device_name(dev)
+        syslog.syslog(f"Input block device is {dev}")
+    except IndexError:
+        syslog.syslog(syslog.LOG_ERR, "Provide block device i.e. xvdf")
+
+    # Convert dev to mapping format
+    dev = convert_dev(dev)
+
+    # Get IMDSv2 token
+    token = get_imdsv2_token()
+
+    # Get instance ID
+    instance_id = requests.get(
+        "http://169.254.169.254/latest/meta-data/instance-id",
+        headers=token,
+        timeout=METADATA_REQUEST_TIMEOUT,
+    ).text
+
+    # Get region
+    region = requests.get(
+        "http://169.254.169.254/latest/meta-data/placement/availability-zone",
+        headers=token,
+        timeout=METADATA_REQUEST_TIMEOUT,
+    ).text
+    region = region[:-1]
+
+    # Parse configuration file to read proxy settings
+    proxy_config = parse_proxy_config()
+
+    # Configure the AWS CA bundle.
+    # In US isolated regions the dedicated CA bundle will be used.
+    # In any other region, the default bundle will be used (None stands for the default settings).
+    # Note: We want to apply a more general solution that applies to every region,
+    # but for the time being this is enough to support US isolated regions without
+    # impacting the other ones.
+    ca_bundle = f"/etc/pki/{region}/certs/ca-bundle.pem" if region.startswith("us-iso") else None
+
+    # Connect to AWS using boto
+    ec2 = boto3.client("ec2", region_name=region, config=proxy_config, verify=ca_bundle)
+
+    # Print the volume id of the device volume
+    get_device_volume_id(ec2, dev, instance_id)
```

Comment:
I'd say we should have:
```python
volume_id = get_device_volume_id(ec2, dev, instance_id)
print(volume_id)
```

because the name of the function is `get_device...` so the scope is to retrieve it, the fact we're printing it it's instead part of the scope of the `main` function.